File: cpp/calculator.cpp
#include <iostream>
using namespace std;
int main()
{
    double n1,n2;
    char op;
    cout <<"Enter first number:";
    cin >> n1;
    cout <<"Enter the operator:";
    cin >>op;
    cout <<"Enter second number:";
    cin >>n2;
    switch(op)
    {
        case '+':
            cout << n1 << '+' << n2 << '=' << n1+n2 << endl;
            break;
        case '-':
            cout << n1 << '-' << n2 << '=' << n1-n2 << endl;
            break;
        case '*':
            cout << n1 << '*' << n2 << '=' << n1*n2 << endl;
            break;
        case '/':
            if(n2!=0)
            {
                cout << n1 << '/' << n2 << '=' << n1/n2 << endl;
            }
            else
            {
                cout << "Invalid Input...ZeroDivisionError";
            }
            break;
        default:
            cout << "Invalid Operation or Number";
            break;
    }
}

File: java/Calculator.java
import java.util.Scanner;
public class Calculator
{
    /**
     * @param args
     */
    public static void main(String[] args)
    {
        int res=0;
        System.out.println("Enter a and b:");
        Scanner s=new Scanner(System.in);
        int a=s.nextInt();
        int b=s.nextInt();
        System.out.println("Enter Operator:");
        char op=s.next().charAt(0);
        switch(op)
        {
            case '+':
                res=a+b;
                break;
            case '-':
                res=a-b;
                break;
            case '*':
                res=a*b;
                break;
            case '/':
                if(b!=0)
                {
                    res=a/b;
                    break;
                }
                else
                {
                    System.out.println("Invalid Input....\nZeroDivisionError");
                }
            default:
                System.out.println("Invalid Input....");
        }
        System.out.println("Result="+res);
        s.close();
    }
}
